gradle-plugins/src/main/groovy/shared.java.vro-dep-constraints.gradle

```diff
@@ -46,12 +46,18 @@ dependencies {
             because("security vulnerability in prior versions")
         }
 
+        // For Kafka svc-bie-kafka service override
+        implementation('org.xerial.snappy:snappy-java:1.1.10.1') {
+            because("security vulnerability in prior versions")
```

Comment:
The `because()` isn't particularly useful since all the constraints in this block resulted from the same reason: SecRel alerts. I recommend leaving it out or referencing the vulnerability ID or CVE ID provided by Aqua -- feel free to bring it up for a team discussion.

Reply:
hmm my thoughts were to include this so it's clear that we cannot go backwards and why it is important that there are no prior versions included in the project. not super attached to it though

Reply:
We can add that as a code comment at the beginning of the constraints block so we don't repeat the same message for each dependency.